client/blocks/color-scheme-picker/docs/example.jsx

```diff
@@ -0,0 +1,43 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React, { PureComponent } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import Notice from 'components/notice';
+import ColorSchemePicker from 'blocks/color-scheme-picker';
+
+class ColorSchemePickerExample extends PureComponent {
+	static displayName = 'ColorSchemePicker';
+
+	state = {
+		selectedColorScheme: null,
+	};
+
+	handleColorSchemeSelection = colorScheme => {
+		this.setState( { selectedColorScheme: colorScheme } );
+	};
+
+	render() {
+		return (
+			<Card>
+				<ColorSchemePicker
+					temporarySelection={ true }
```

Comment:
`={ true }` can be omitted here since it will be inferred by the prop being present without a value
```jsx
<ColorSchemePicker
  temporarySelection
```